src/helpers/spaces.ts

```diff
@@ -145,6 +145,23 @@ async function getVotes() {
   return await db.queryAsync(query);
 }
 
+export async function getUniqueVotersForSpace(spaceId) {
+  const query = `
+    SELECT space, GROUP_CONCAT(DISTINCT voter) AS unique_voters
+    FROM votes
+    WHERE space = ?
+    GROUP BY space
+  `;
```

Comment:
This could be simplified to:
```suggestion
  const query = `SELECT DISTINCT(voter) FROM votes WHERE space = ?`;
```
I think you may want to add a limit here too because in some space there can be millions of voters, which would take too long to resolve.